### ISSUE TITLE
Hyperlinks to sections which have spaces in their names are not recognised as hyperlinks

### ISSUE THREAD

**Issue by Blackilykat** [silverbulletmd/silverbullet#1106]
when trying to hyperlink a section with a space in it `[just](#Like this)`, it's no longer recognised as a hyperlink and the first word is recognised as a tag
![image](https://github.com/user-attachments/assets/3336fdcf-4f3e-4c25-8d05-984c4a9bc4a0)

**Comment by zefhemel** [comment]
Right. I think this follows markdown standards. You have to put < and > around urls with spaces like `[hello](<#hello there>)`.

**Comment by Blackilykat** [comment]
> You have to put < and > around urls with spaces like `[hello](<#hello there>)`.
That fixes it, thanks! But when typing out the `#` the autocomplete suggests the text of the section, but doesn't automatically add those. It should probably do that
